autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.css

```diff
@@ -14,6 +14,20 @@
 .schedule-container {
     flex-grow: 1;
     flex-basis: 0;
+    height: calc(100% - 8px);
```

Comment:
```suggestion
    height: 100%;
```
The current calculation has too much bottom padding
| 100% | 100% - 8px |
|-|-|
| ![image](https://user-images.githubusercontent.com/28655462/121814438-bc09ef00-cc36-11eb-9643-b25381076d34.png) | ![image](https://user-images.githubusercontent.com/28655462/121814440-c1ffd000-cc36-11eb-81f6-a09d06bb0a22.png) |

Reply:
Am I missing something? I did this intentionally - this bottom padding makes it line up with the bottom padding of the middle column and the course card column.

|100%|100% - 8 px|
|--|--|
|![chrome_7VK7tqDy9M](https://user-images.githubusercontent.com/7295783/121940299-c8f41480-cd02-11eb-99e6-9b0492b25f91.png)|![chrome_pjxp3LkOqc](https://user-images.githubusercontent.com/7295783/121940313-cc879b80-cd02-11eb-912e-695776686583.png)|